### DISCUSSION TITLE
Have a pane to list all objects and their metadata?

### DISCUSSION THREAD

**Discussion by Chris-ECE** [jgraph/drawio#4094] [Ideas]
For large class diagrams, it would be extremely useful to be able to open a pane that alphabetically lists all of the classes in the diagram, along with their metadata. - And even better, if that list was:-

1. Exportable, as an excel/csv table, to provide a glossary for the diagram; and
2. Editable, so that you could update the names and metadata of all those classes in your diagram (to eliminate the worry that you might have missed one out when doing an update to the diagram).

At the moment, I am having to add metadata to my classes one-by-one using the Ctrl+M command, which is taking ages. I have considered exporting and directly editing the XML version of my diagram, but this is an even more daunting task.

Thankyou

**Comment by unece-stat** [comment]
I agree that this would be a very useful feature. - Although there is a plugin called "props" which displays metadata when the mouse is hovered over the item, this can only display metadata one item at a time, so is quite limited.

In addition to the features you propose, I would also add the ability to import metadata into a diagram somehow.
